msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/brokerapi/TestCase1561087.java

```diff
@@ -38,6 +39,7 @@
 // Flight settings
 // https://identitydivision.visualstudio.com/Engineering/_workitems/edit/1561087
 @SupportedBrokers(brokers = BrokerHost.class)
+@LocalBrokerHostDebugUiTest
```

Comment:
Isn't supported brokers annotation doing similar thing?

Reply:
Yes, the issue is that from what I researched you can't use @SupportedBrokers with a particular value for "brokers" when assigning test targets (i.e. you can't assign a test run to ignore tests that have the SupportedBrokers annotation with value BrokerHost.class). To be able to create this test target distinction, we need an annotation specifically for tests that requires msal test app built with Local BrokerHost Debug, hence adding this new annotation